client/packages/invoices/src/OutboundShipment/DetailView/Footer/Footer.tsx

```diff
@@ -93,6 +96,8 @@ export const FooterComponent: FC<FooterComponentProps> = ({
     if (selectedUnallocatedEmptyLines.length !== 0) {
       getConfirmation();
     } else onAllocate();
+
+    clearSelected();
```

Comment:
I think this should await allocation before clearing? Keep selection if there's an error?

Reply:
Have removed. Was already clearing